src/main/java/sirius/web/templates/pdf/SemaphoreContext.java

```diff
@@ -8,34 +8,42 @@
 
 package sirius.web.templates.pdf;
 
+import sirius.kernel.async.SubContext;
+
 import java.util.concurrent.Semaphore;
 
 /**
  * This class is used to limit the number of concurrent resource handling operations.
  * <p>
  * This is used to avoid overloading the system with too many concurrent resource handling operations.
```

Comment:
```suggestion
 * This class is used to limit the number of concurrent resource handling operations per belonging CallContext.
 * <p>
 *  The usecase is that we want to control the number of spawned threads to avoid overloading the 
 *  system with too many concurrent operations.
```